guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -367,7 +333,7 @@ This Content View is used to facilitate the library content import.
 
 |Export a content view version promoted  to the Dev Environment|`hammer content-export complete version --content-view=view --organization="Default_Organization" --lifecycle-environment=’Dev’`
 
-|Export a content view in smaller chunks (200 mb slabs)|`hammer content-export complete version --content-view=view --version=1.0 --organization="Default_Organization" --chunk-size-mb=200`
+|Export a content view in smaller chunks (200 mb slabs)|`hammer content-export complete version --content-view=view --version=1.0 --organization="Default_Organization" --chunk-size-gb=2`
```

Comment:
Does `200 mb slabs` also have to be changed?

Reply:
thanks!